lib/node_modules/@stdlib/stats/base/dists/planck/median/benchmark/benchmark.js

```diff
@@ -32,19 +31,14 @@ var median = require( './../lib' );
 
 bench( pkg, function benchmark( b ) {
 	var lambda;
-	var len;
 	var y;
 	var i;
 
-	len =100;
-	lambda = new Float64Array( len );
-	for ( i = 0; i < len; i++ ) {
-		lambda[ i ] = uniform( 1.0, 10.0);
-	}
+	lambda = uniform( 100, 1.0, 10.0 ); 
```

Comment:
```suggestion
	lambda = uniform( 100, 0.1, 10.0 );
```

Something that doesn’t make a huge difference but would be better to have the same range as the original for consistency. :)

Reply:
absolutely! had suggested something similar in this [suggestion](https://github.com/stdlib-js/stdlib/pull/5314#discussion_r1963528446) must have been overlooked.